book/types_of_data.md

```diff
@@ -128,28 +130,26 @@ Nushell also has a special type for file sizes. Examples include `100b`, `15kb`,
 
 The full list of filesize units are:
 
-- `b`: bytes
-- `kb`: kilobytes (aka 1000 bytes)
-- `mb`: megabytes
-- `gb`: gigabytes
-- `tb`: terabytes
-- `pb`: petabytes
-- `eb`: exabytes
-- `kib`: kibibytes (aka 1024 bytes)
-- `mib`: mebibytes
-- `gib`: gibibytes
-- `tib`: tebibytes
-- `pib`: pebibytes
-- `eib`: exbibytes
+| Decimal (factor of 1000) | Binary (factor of 1024) |
```

Comment:
I like the table other than `kB` looks weird below but I'd also lead with unit case doesn't matter. kib = KiB = kIB = Kib, etc.